new_module.py

```diff
@@ -1,13 +1,18 @@
+from __future__ import annotations
+
 class Genius:
-    def __int__(self):
-        self._memory = -1
+    def __init__(self, memory: int | None = None):
+        self._memory = memory
 
-    def get_memory(self):
+    def get_memory(self) -> int | None:
         return self._memory
 
     def set_memory(self, number: int) -> None:
         self._memory = number
```

Comment:
make a normal getter and setter out of these methods with property decorator
also add clear() method which makes self.memory equal to 0